config/environment.js

```diff
@@ -23,6 +23,12 @@ module.exports = function(environment) {
     }
   };
 
+  ENV['ember-a11y-testing'] = {
+    componentOptions: {
+      turnAuditOff: true, // Change to run a11y audit on all dev build and serves
```

Comment:
should we turn on instead of adding the explicit test cases.  I don't have super intimate knowledge of the addon so unsure if its an equal substitute.